### ISSUE TITLE
Support editing of the alt attribute inside the image dialog

### ISSUE THREAD

**Issue by jnnkB** [GetmeUK/contenttools-2-roadmap#12]
I think this would be very helpful for building websites with good accessibility – which you should. 

Currently it's only possible after you uploaded the image and most user's won't notice it's possible or forget it.

**Comment by anthonyjb** [comment]
Yep - agreed - this is possible in the current manhattan integration I have for this reason. Images will be vastly improved overall. I've got picture support in at the moment and I'm currently (this week) adding in support for multiple sources. This is an early stop gap for manhattan CMS, but it will be a useful proof of concept for v2 of CT.
